backend/app-orchestrator-service/api/src/main/kotlin/api/Jobs.kt

```diff
@@ -475,6 +482,14 @@ object Jobs : CallDescriptionContainer("jobs") {
         }
     }
 
+    val utilization = call<JobsUtilizationRequest, JobsUtilizationResponse, CommonErrorMessage>("utilization") {
+        httpUtilization(baseContext)
```

Comment:
See my other comment.

```suggestion
        httpRetrieve(baseContext, "utilization")
```